operator-framework-core/src/main/java/io/javaoperatorsdk/operator/AggregatedOperatorException.java

```diff
@@ -2,18 +2,19 @@
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.Objects;
 
 public class AggregatedOperatorException extends OperatorException {
 
   private final List<Exception> causes;
 
   public AggregatedOperatorException(String message, Exception... exceptions) {
-    super(message, exceptions[0]);
+    super(message, Objects.requireNonNull(exceptions)[0]);
```

Comment:
That doesn't help, actually, since this still will throw an NPE if `exceptions` isn't present. With varargs, it's a definite possibility, though, granted, creating an `AggregatedOperatorException` and not passing any exceptions is probably expected to fail…